### PR TITLE
HCALDQM: Fix digi size reference for summary plots (10_1_X)

### PR DESCRIPTION
**Pull request by DryRun**
Fixes the digi size reference (HB, HE from 10 to 8) for the summary plots. Also moves the location of the reference from C++ (HcalCommon/interface/Constants.h) to python (DigiTask.py, and copied via import to HcalOnlineHarvesting.py and HcalOfflineHarvesting.py).

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

File: DQM/HcalTasks/python/DigiTask.py [hunk 1]
```diff
@@ -27,7 +27,15 @@
 	thresh_unifh = cms.untracked.double(0.2),
 
 	qie10InConditions = cms.untracked.bool(False),
+
+	# Reference digi sizes
+	refDigiSize = cms.untracked.vuint32(10, 10, 10, 4), # HB, HE, HF, HO
```
Comment (deguio) [inline DQM/HcalTasks/python/DigiTask.py]: the comment that says `HB, HE, HF, HO` should be `HB, HE, HO, HF`, right?

**Comment by mariadalfonso** [comment]
@jfernan2 can we start the test ?

**Comment by jfernan2** [comment]
please test

**Comment by dmitrijus** [comment]
+1

The error seems unrelated to this PR.

```
Begin processing the 60th record. Run 1, Event 60, LumiSection 1 on stream 0 at 20-Mar-2018 12:04:36.605 CET
%MSG-e MissingProduct:  HybridClusterProducer:cleanedHybridSuperClusters  20-Mar-2018 12:04:37 CET Run: 1 Event: 60
could not get a handle on the EcalRecHitCollection!
%MSG
%MSG-e MissingProduct:  HybridClusterProducer:uncleanedHybridSuperClusters  20-Mar-2018 12:04:37 CET Run: 1 Event: 60
could not get a handle on the EcalRecHitCollection!
%MSG
----- Begin Fatal Exception 20-Mar-2018 12:04:37 CET-----------------------
An exception of category 'fastsim::ParticleManager' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 60 stream: 0
   [1] Running path 'simulation_step'
   [2] Calling method for module FastSimProducer/'fastSimProducer'
Exception Message:
unknown pdg id: 9010221
----- End Fatal Exception -------------------------------------------------
%MSG-w PFEGammaAlgo::mergeROsByAnyLink:  PFEGammaProducer:particleFlowEGamma  20-Mar-2018 12:04:37 CET Run: 1 Event: 57
Need to implement proper merging of two gsf candidates!
%MSG
```

**Comment by fabiocos** [comment]
please test

**Comment by fabiocos** [comment]
+1

**Pull request merged**